src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/IndexManagementPlugin.kt

```diff
@@ -59,32 +62,27 @@ import org.elasticsearch.threadpool.ThreadPool
 import org.elasticsearch.watcher.ResourceWatcherService
 import java.util.function.Supplier
 
-internal class IndexStateManagementPlugin : JobSchedulerExtension, ActionPlugin, Plugin() {
+internal class IndexManagementPlugin : JobSchedulerExtension, ActionPlugin, Plugin() {
 
     private val logger = LogManager.getLogger(javaClass)
-    lateinit var indexStateManagementIndices: IndexStateManagementIndices
+    lateinit var indexManagementIndices: IndexManagementIndices
     lateinit var clusterService: ClusterService
 
     companion object {
-        const val PLUGIN_NAME = "opendistro-ism"
-        const val ISM_BASE_URI = "/_opendistro/_ism"
+        const val PLUGIN_NAME = "opendistro-im"
+        const val OPEN_DISTRO_BASE_URI = "/_opendistro"
+        const val ISM_BASE_URI = "$OPEN_DISTRO_BASE_URI/_ism"
         const val POLICY_BASE_URI = "$ISM_BASE_URI/policies"
-        const val INDEX_STATE_MANAGEMENT_INDEX = ".opendistro-ism-config"
-        const val INDEX_STATE_MANAGEMENT_JOB_TYPE = "opendistro-managed-index"
+        const val INDEX_MANAGEMENT_INDEX = ".opendistro-ism-config"
+        const val INDEX_MANAGEMENT_JOB_TYPE = "opendistro-index-management"
```

Comment:
As a note: Job Scheduler only lets us specify one job type which is why we changed this to be more generic for the whole plugin. At the same time it does not actually use this so it's not that important.